src/api/services/participantsService.ts

```diff
@@ -73,6 +73,10 @@ export const getAttachedSiteIDs = async (): Promise<SiteIdType[]> => {
   return sites.map((s) => s.siteId);
 };
 
+export const getApprovedParticipant = async (): Promise<Participant[]> => {
```

Comment:
maybe just name this `getParticipantsApproved` so we don't rename the import in `participantsRouter`?

Reply:
Alternatively we could just call it `getParticipants`? We already differentiate it from the ones awaiting approval. What do you think?

Reply:
I think keeping it as approved is probably better. There's some chance someone might want a function that just returns all participants.